source/split.d.ts

```diff
@@ -22,8 +22,14 @@ array = split(items, ',');
 export type Split<
 	S extends string,
 	Delimiter extends string,
+> = SplitHelper<S, Delimiter>;
+
+export type SplitHelper<
```

Comment:
```suggestion
type SplitHelper<
```

I don't think it should be exported.

Reply:
Yeah, removed!